openapi/controllers/apijsonrequest.py

```diff
@@ -39,16 +40,16 @@ def __init__(self, *args):
         if jsonp and self.httprequest.method == 'POST':
             # jsonp 2 steps step1 POST: save call
             def handler():
-                self.session['jsonp_request_%s' % (request_id,)] = self.httprequest.form['r']
+                self.session['jsonp_request_%s' % (request_id,)] = self.httprequest.form['rb']
```

Comment:
This part of code was made by @blaggacao and I don't know about it much. Could you explain a little bit where ``'r'`` and ``'rb'`` are come from?

Reply:
> In Python 3, reading files in r mode means decoding the data into Unicode and getting a str object. Reading files in rb mode means reading the data as is, with no implicit decoding, and saving it as bytes.

I think literal strings are unicode by default in Python3?

Reply:
My question was more about the variable ``httprequest.form``. Do you know anything about it? I cannot find an explanation why it should have `'r'` or `'rb'` item

Reply:
No I don't. I just fixed errors we ran into while trying to setup the module.